### DISCUSSION TITLE
How can i get 'Galaxy Brain' achievement

### DISCUSSION THREAD

**Discussion by t0linixer** [juddydev/react-3D#5] [Q&A] [answered]
I have a question
How can i get 'Galaxy Brain' achievement

**Comment by alpine-storm** [comment, marked as answer]
to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
Hope this answer has helped you, and all the best in getting the achievement! :)
Reference: https://github.com/orgs/community/discussions/18467

**Reply by t0linixer** [reply]
thank you
